applications/GeoMechanicsApplication/python_scripts/geomechanics_analysis.py

```diff
@@ -5,7 +5,9 @@
 sys.path.append(os.path.join('..','..','..'))
 
 import KratosMultiphysics as Kratos
+# Linear solver is not working as expected
 import KratosMultiphysics.LinearSolversApplication
+import KratosMultiphysics.ExternalSolversApplication
```

Comment:
```suggestion
```
this is deprecated and will be removed in the future

Reply:
I have some problems with LinearSolvers when I use cable elements derived from StructuralMechanicsApplication. The solver gives "singular matrix", while the same example can be solved successfully with the ExternalSolvers.

Reply:
see my comment below

which solvers did you use?
have you tried with the pardiso?

Reply:
Ah I would still remove this, as can be done through json as explained

Reply:
Thanks @philbucher. I removed it as well. By the way, how can I use Pardiso solver in Kratos. Is there an example?